internal/connect/utils.go

```diff
@@ -10,15 +10,15 @@ import (
 	"github.com/confluentinc/cli/v3/pkg/errors"
 )
 
-func getConfig(cmd *cobra.Command) (*map[string]string, error) {
+func getConfig(cmd *cobra.Command, isUpdate bool) (*map[string]string, *[]map[string]any, error) {
```

Comment:
It looks like this function is doing two separate things. Can we split this into two functions: `getConfig()` and `getOffsets()`?

Reply:
both is read from same file, i can rename the method.

Reply:
Done